website/docs/docs/guides/migration-guide/migrating-from-spark-to-databricks.md

```diff
@@ -0,0 +1,109 @@
+
+When an admin changes the dbt Cloud's connection to use the dbt-databricks adapter instead of the dbt-spark adapter, your team will not lose their credentials. This makes migrating from dbt-spark to dbt-databricks straightforward as it only requires deleting the connection and re-adding the cluster/endpoint information. Both the admin and users of the project need not re-enter personal access tokens.
+
+#### Procedure
+
+An admin of the dbt Cloud project running on Databricks should take the following steps to migrate from using the generic Spark adapter to the Databricks-specfic adapter. This should not cause any downtime for production jobs, but we recommend that you schedule the connection change when there is not heavy IDE usage for your team to avoid disruption.
+
+1. Select Account Settings in the main navigation bar.
+2. On the Projects tab, scroll until you find the project you'd like to migrate to the new dbt-databricks adapter.
+3. Click the hyperlinked Connection for the project.
+4. Click the "Edit" button in the top right corner.
+5. Click "Delete Connection" at the bottom of the page, and then confirm ythat you'd like to delete the connection.
+
+    > :warning:, the only information that will be deleted is the cluster or endpoint host and id. Your credential will still be saved. See above note.
+5. After being brought back to the project overview page, click "Configure a connection" so that you can start the connection flow for the new dbt-databricks adapter.
+6. Select the Databricks adapter that is not listed as "to be depecated", and enter:
+    1. the `hostname`
+    2. `http_path`, and
+7. Click save.
+
+After the above steps have been performed, all users will have to refresh their IDE before being able to start working again. It should complete in less than a minute. 
+
+
+
+
+
+### dbt Core
+
+In dbt Core, migrating to the dbt-databricks adapter from dbt-spark requires that you:
+1. install the new adapter in your environment, and
+2. modify your target in your `~/.dbt/profiles.yml`
+
+These changes will be needed for all users of your project.
+
+#### Example
+
+If you're using `dbt-spark` today to connect to a Databricks SQL Endpoint, the below examples show a good before and after of how to authenticate. The cluster example is also effectively the same.
+
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+your_profile_name:
+  target: my_databricks
+  outputs:
+    dev:
+      type: spark
+      method: odbc
+      driver: '/opt/simba/spark/lib/64/libsparkodbc_sb64.so'
```

Comment:
Do we need to specify this?

Reply:
@allisonwang-db this is how a user who is currently using Databricks via dbt-spark would have their profile specified. Fortunately, it becomes much more simple for dbt-databricks